tests/torch_datasets/test_site.py

```diff
@@ -119,20 +100,49 @@ def test_convert_from_dataset_to_dict_datasets(sites_dataset):
 
 
 def test_site_dataset_with_dataloader(sites_dataset) -> None:
-    expected_coods = {
-        "site__date_cos",
-        "site__time_cos",
-        "site__time_sin",
-        "site__date_sin",
-    }
-
-    dataloader = DataLoader(sites_dataset, collate_fn=None, batch_size=None)
-
-    sample = next(iter(dataloader))
+    batch_size = 2
+
+    dataloader = DataLoader(
```

Comment:
do we expect to use batch_size when making samples? If not, should role back to batch_size=None, and collate_fn=None, and adjust the code below